docs/iceberg-rest-service.md

```diff
@@ -121,25 +121,30 @@ The Gravitino Iceberg REST catalog service uses the memory catalog backend by de
+| `gravitino.iceberg-rest.catalog-backend`                                  | The Catalog backend of the Gravitino Iceberg REST catalog service. Use the value **`hive`** for a Hive catalog.                                                                                                                                                | `memory`                                                                       | Yes                                                                                | 0.2.0         |
+| `gravitino.iceberg-rest.uri`                                              | The Hive metadata address, such as `thrift://127.0.0.1:9083`.                                                                                                                                                                                                  | (none)                                                                         | Yes                                                                                | 0.2.0         |
+| `gravitino.iceberg-rest.warehouse`                                        | The warehouse directory of the Hive catalog, such as `/user/hive/warehouse-hive/`.                                                                                                                                                                             | (none)                                                                         | Yes                                                                                | 0.2.0         |
+| `gravitino.iceberg-rest.catalog-backend-name`                             | The catalog backend name passed to underlying Iceberg catalog backend. Catalog name in JDBC backend is used to isolate namespace and tables.                                                                                                                   | `hive` for Hive backend, `jdbc` for JDBC backend, `memory` for memory backend  | No                                                                                 | 0.5.2         |
+| `gravitino.iceberg-rest.authentication.type`                              | The type of authentication for Iceberg rest catalog backend. This configuration only applicable for for Hive and REST backend, and only supports `Kerberos`, `simple` currently. As for JDBC backend, only username/password authentication was supported now. | `simple`                                                                       | No                                                                                 | 0.6.0         |
```

Comment:
there is no REST catalog backend for Iceberg REST server

Reply:
OK